Pages/Volunteers.razor

```diff
@@ -55,4 +63,22 @@ else
     {
         m_uriHelper.NavigateTo($"event/{id}");
     }
+
+    public void NavigateToAddVolunteer()
+    {
+        m_uriHelper.NavigateTo($"addvolunteer");
```

Comment:
This `$` is not needed here
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/interpolated

Reply:
Removed the $.